tutorials/exposing-asp.net-webapi-using-.net-core-with-cloud-functions/index.md

```diff
@@ -0,0 +1,211 @@
+           {
+               c.SwaggerEndpoint("/swagger/v1/swagger.json", "My API V1");
+           });
+
+           app.UseMvc();
+        }
+
+## Step 3 - Create OpenAPI specification for our API
+
+1. Press `F5` in VS Code. Browse to [http://localhost:5000/swagger/v1/swagger.json](http://localhost:5000/swagger/v1/swagger.json)
+
+    You should see Open API specification for our APIs in JSON format. However we will work with YAML representation of the same JSON. Even though Google Endpoints supports both json and YAML representations of OpenAPI, YAML is much more human friendly, so in case we will want to make manual changes to the specification, doing it in YAML file will be easier.
+
+2. Copy the JSON content from the browser and paste it into [https://editor.swagger.io](https://editor.swagger.io). 
+    You will be asked whether to translate JSON to YAML. Say `YES`. You will see this:
+    ![Swagger Editor](https://storage.googleapis.com/gcp-community/tutorials/exposing-asp.net-webapi-using-.net-core-with-cloud-functions/SwaggerYamlFromEditor.io.png)
+
+3. Create a new file in VS Code under the main folder `ExposeApiWithEndpointsCore`. Name it `openapi.yaml` and paste the content of the generated YAML from https://editor.swagger.io.
+
+    Now we are ready to deploy the API to Google Cloud Platform!
+
+## Step 4 - Create a new GCP Project
+
+1. [Create a new project](https://cloud.google.com/resource-manager/docs/creating-managing-projects#creating_a_project) at https://console.cloud.google.com
+
+    We will create App Engine Flexible Environment and deploy our API to it from the command line. So now we need to make sure that Google Cloud SDK is pre-configured to work with the project we just created. 
+
+2. Open Google Cloud SDK command line in elevated mode (Administrator), and type:
+
+    2.1 Update SDK components to the latest
+
+        gcloud components update
+
+    2.2 Make sure that gcloud has permissions to work with our cloud project.
+
+        gcloud auth login
+
+    This will open a new browser window where we need to authenticate using our user account.
+
+    2.3 Configure default project for all subsequent commands we will execute:
+
+        gcloud config set project `YOUR_PROJECT_ID`
+
+    2.4 Create [App Engine flexible environment](https://cloud.google.com/appengine/docs/flexible/) to which we will deploy our API. You can find the list of supported regions [here](https://cloud.google.com/appengine/docs/locations).
+
+        gcloud app create --project --region=`YOUR_REGION`
+
+    Congrats! We just finished with cloud setup! Now it's time to deploy the API specification and API implementation.
+
+## Step 5 - Deploy API Specification to Google Endpoints
+
+1. Open `openapi.yaml` that was created in step 3 in VS Code.
+
+2. Add `host` declaration right before the `paths` section. The value should be `YOUR-PROJECT-ID.appspot.com` (replace `YOUR-PROJECT-ID` with your cloud project Id):
+    <pre>
```

Comment:
For security reasons, all HTML and anything that is HTML-like is stripped during publishing. To get around this, move HTML and HTML-like snippets into their own files, and then in your tutorial link to the files. Here's an example of a tutorial that does this: https://github.com/GoogleCloudPlatform/community/tree/master/tutorials/appengine-dropwizard

I see at least two places in this tutorial where HTML-link stuff would be stripped out.